cdap-cli/src/main/java/co/cask/cdap/cli/command/GetProgramRunsCommand.java

```diff
@@ -78,12 +78,13 @@ public void perform(Arguments arguments, PrintStream output) throws Exception {
     }
 
     Table table = Table.builder()
-      .setHeader("pid", "end status", "start", "stop")
+      .setHeader("pid", "end status", "start", "run", "stop")
```

Comment:
It seems a bit confusing to have `run start time`, since the last descriptor is `start`. Do you think that's okay?